G1ANT.Chrome.Driver/ChromeClient.cs

```diff
@@ -0,0 +1,67 @@
+﻿using G1ANT.Browser.Driver.Actions;
+using G1ANT.Browser.Driver.Data;
+using G1ANT.Browser.Driver.Services;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace G1ANT.Chrome.Driver
+{
+    public class ChromeClient : BrowserClient
```

Comment:
ChromiumClient?

Reply:
Will be added during Edge and Brave implementation